### ISSUE TITLE
Missing git tag for version 2.3.4

### ISSUE THREAD

**Issue by pmorch** [jbaysolutions/vue-grid-layout#338]
There are git tags for many other versions, just not for 2.3.4. Which might be nice for posterity.

**Comment by gmsa** [comment]
forgot about that, thanks

**Comment by pmorch** [comment]
Just FYI: I think that 2.3.4 as released on npm was actually 514de88, not 9e13ea9.

The diff is a single line to the README.md, but I noticed it and just wanted to post it. I don't expect any action.

[npm version 2.3.4](https://www.npmjs.com/package/vue-grid-layout/v/2.3.4) doesn't contain  commit 8a5fb48fd4c10ac28b3c4a6feb08d52657a8958a which is a change to the README.md:

```
commit 8a5fb48fd4c10ac28b3c4a6feb08d52657a8958a
Author: Cristian Garcilazo <30292133+Cgarcilazo@users.noreply.github.com>
Date:   Fri Mar 8 15:33:57 2019 -0300

    Add Data Providers to readme

diff --git a/README.md b/README.md
index 255170a..9bf185b 100644
--- a/README.md
+++ b/README.md
@@ -39,6 +39,7 @@ Also check https://cli.vuejs.org/guide/build-targets.html#library
 
 - [Draxed](https://www.draxed.com/?utm_source=github&utm_medium=web&utm_campaign=vue-grid-layout)
 - [cryptotiles](https://www.cryptotiles.io/?utm_source=github&utm_medium=web&utm_campaign=vue-grid-layout)
+- [Data Providers](https://www.dataproviders.io/?utm_source=github&utm_medium=web&utm_campaign=vue-grid-layout)
 
 *Know of others? Create a PR to let me know!*

```
